components/FlexibleHeader/tests/unit/FlexibleHeaderNoTrackingScrollViewTests.swift

```diff
@@ -1,26 +0,0 @@
-// Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import XCTest
-import MaterialComponents.MaterialFlexibleHeader
-
-// Tests confirming contract with no tracking scroll view
-class FlexibleHeaderNoTrackingScrollViewTests: FlexibleHeaderTrackingScrollViewTests {
-
-  override func setUp() {
-    super.setUp()
-
-    view.trackingScrollView = nil
-  }
-}
```

Comment:
Just want to confirm this file isn't being imported anywhere right?

Reply:
It didn't cause any problems to delete it and I can't find any mentions of it in the code base.